RULES.md

```diff
@@ -0,0 +1,150 @@
+
+- File names should be the name of the dish with words separated by hyphens (`-`), not underscores, and definitely not spaces.
+
+#### Tags
+
+- Recipes should end with a list of relevant tags.
+The syntax is:
+
+```md
+;tags: tag1 tag2 tag3
+```
+
+- The tag line should be a single line, at the end of the markdown file, preceded by a blank line.
+- Add between a few (1-4) tags.
+- **Prioritize existing tags**.
+- As a general guideline, add the country from which the recipe originates if the recipe is representative of said country, using the adjective form (eg. *mexican*, *italian*, etc).
+- Tag the main ingredient if it's something slightly special.
+
+##### List of special, categorical tags to use if relevant
+
+- `basic`: for basic recipes that aren't meant to be standalone but are supposed
+  to be incorporated in another recipe.
+- `breakfast`
+- `cheesefare`: for recipes that include dairy but no milk.
+- `dessert`
+- `drink`
+- `quick`: for recipes that can be cooked in under ~20 minutes.
+- `side`: side dishes such as mash, fries, etc.
+- `snack`
+- `spread`
+- `strict-fast`: for recipes that contain no meat or dairy.
+- `vegan`
+- `vegetarian`
+
+#### Images
+
+- Images are stored in `data/pix`.
+- Each recipe can have a title image at the top and perhaps several instructional images as absolutely necessary.
+- Do not add stock images you found on the internet.
+Take a good picture yourself of the actual dish you prepared.
+- If you see a bad or substandard image, you may submit a better one.
+- Images should be in `.webp` format and with as small a file size as possible.
+If you submit an image for say, `chicken-parmesan.md`, it should be added as `pix/chicken-parmesan.webp`.
+- If you would like to add additional directional images, they should be numbered with two digits like `pix/chicken-parmesan-00.webp`, `pix/chicken-parmesan-01.webp`, etc.
+
+### Bugmanism is banned
+
+Vegan or vegetarian recipes are fine, but specific bugman food imitating authentic food is disallowed.
+
+You could always substitute any ingredient for any other ingredient, or follow any instruction any way you like whilst you are preparing the dish, for example to swap meat out for a substitute.
+However, do not propose these personal changes to [based.cooking](https://based.cooking).
+
+#### Examples
+
+- Never use a meat substitute instead of real meat.
+Non-meat pretending to be meat, like soy in the shape of sausages, or mycoprotein meat substitutes, are banned.
+- Tofu is only allowed when used in an authentic dish, and not as a meat substitute.
+
+### No soyboys
+
+No soydev code, Javascript, WebAssembly, or other bloat is permitted.
+
+Soybeans and soy-based food are acceptable.
+
+### No meme-tier sweet or unhealthy dishes
+
+Only substantial dishes are allowed, no hyper-sugary desserts or junk food.
+
+### Treat your audiance as if they are intelligent
+
+- Do not include ubiquitous ingredients like salt and pepper in the ingredients list.
```

Comment:
I have seen some exceptions to this... Some people have submitted recipes with a specific amount of salt which is baked into the dish (ie not added for seasoning after the fact) and I think at least one has been merged.

So if we go with that, we can add that salt is allowed if a specific quantity is given, and it isn't used for taste at the end of the dish and is baked (literally) into the dish, like bread for example

Reply:
Fixed in recent commit.